brands/shop/furniture.json

```diff
@@ -182,6 +182,15 @@
       "shop": "furniture"
     }
   },
+  "shop/furniture|Freedom Furniture": {
```

Comment:
A `matchNames` of “Freedom Furniture” could be added here, since the `name` is only “Freedom”.

Reply:
good catch - fixed this in 00bc22fd